docs/docs/transforms/label.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: transform
+title: Label Transform
+permalink: /docs/transforms/label/index.html
+---
+
+The **label** transform repositions texts in text mark, so that their placements are not colliding with other elements in the chart. Those elements include marks that have name included in property `avoidMarks` and the mark that is used as backing data (as explained in [reactive geometry](../../marks/#reactivegeom)) of the text mark this label transform is transforming.
```

Comment:
```suggestion
The **label** transform repositions text marks so that they are not colliding with other elements in the chart. The label transform avoids all marks of types listed in `avoidMarks` and the mark that is used as backing data (as explained in [reactive geometry](../../marks/#reactivegeom)) of the text mark this label transform is using as input.
```